compiler/src/dotty/tools/dotc/transform/CheckUnused.scala

```diff
@@ -430,6 +430,20 @@ object CheckUnused:
         else
           exists
       }
+
+      // not report unused transparent inline imports
+      for {
```

Comment:
Can we just handle it in `registerImport` by not adding those imports to `unusedImport`?